.env.template

```diff
@@ -1,2 +1,34 @@
+# This is a template file for all environment variables used in Serve.
+# Please copy it to a file called .env and change the settings as you see fit.
```

Comment:
I decided to move all the env variables from the docker compose file to `.env.template` to avoid having to change `DEBUG` and others at multiple places. 

Now we can set IP at one place, and it automatically uses that everywhere. Much easier.